deepgrow/ignite/train.py

```diff
@@ -221,8 +221,8 @@ def create_trainer(args):
     train_handlers = [
         LrScheduleHandler(lr_scheduler=lr_scheduler, print_lr=True),
         ValidationHandler(validator=evaluator, interval=args.val_freq, epoch_level=True),
-        StatsHandler(tag_name="train_loss", output_transform=lambda x: x["loss"]),
-        TensorBoardStatsHandler(log_dir=args.output, tag_name="train_loss", output_transform=lambda x: x["loss"]),
+        StatsHandler(tag_name="train_loss", output_transform=from_engine(["loss"], first=True)),
```

Comment:
are you sure u need this first=True flag?

Reply:
Yes, according to the latest API, in order to `decollate` the `batch` and `output` (https://github.com/Project-MONAI/MONAI/blob/dev/monai/engines/workflow.py#L184), the scalar values will be added the batch dimension via duplicating (https://github.com/Project-MONAI/MONAI/blob/8365443ababac313340467e5987c7babe2b5b86a/monai/data/utils.py#L373).